src/main/webapp/assets/js/reviews.js

```diff
@@ -20,22 +20,31 @@ async function getReviews() {
 function createReviewSection(review) {
   const reviewWrapper = document.createElement("div");
   reviewWrapper.className = "review-card";
-
+  
+  let stars = createStars(review.rating);
   reviewWrapper.innerHTML =
     `<div class="review-card-container">
```

Comment:
Can consider moving to a separate function so its neater?

Reply:
Are you referring to the .innerHTML segment? If so, I'm not sure why it's neater to move that to a separate function? It seems more intuitive to keep the things related to reviewWrapper inside the createReviewSection function...